cogboard-webapp/src/components/widgets/types/ServiceCheckWidget.js

```diff
@@ -62,7 +66,7 @@ const ServiceCheckWidget = props => {
         </WidgetButton>
       </Caption>
       <CaptionWithPointer title={body} onClick={handleClick}>
-        Response: {bodyMessage}
+        ⯆ Response: {bodyMessage}
```

Comment:
⯆ - what is this?

Reply:
Removed (it looks like on some devices don't recognize this sign)